Makefile

```diff
@@ -322,11 +321,16 @@ telemetrygen:
 
 .PHONY: update-otel
 update-otel:$(MULTIMOD)
-	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector -m stable --commit-hash $(OTEL_STABLE_VERSION)
-	git add . && git commit -s -m "[chore] multimod update stable modules"
-	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector -m beta --commit-hash $(OTEL_VERSION)
-	git add . && git commit -s -m "[chore] multimod update beta modules"
+	$(MULTIMOD) sync -s=true -o $(SRC_ROOT) -m stable 
```

Comment:
Is there a reason we're moving from `opentelemetry-collector` being the focus of update here to `openelemetry-collector-contrib` instead?

Reply:
yes, i did it for testing purpose and didn't reverse it, that's also why --commit-hash wasn't working for me
sorry for confusion, should've seen that earlier, when you suggsted the envvar `SRC_ROOT`

i guess we could go with $SRC_PARENT_DIR/opentelemetry-collector or just stay as it was intended

i filled that + the commit-hash + OTEL-Version back in